CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ incremented for features.
 * lang: Add `programdata_address: Option<Pubkey>` field to `Program` account. Will be populated if account is a program owned by the upgradable bpf loader ([#1125](https://github.com/project-serum/anchor/pull/1125))
 * lang,ts,ci,cli,docs: update solana toolchain to version 1.8.5([#1133](https://github.com/project-serum/anchor/pull/1133))
 
+### Breaking
+
+* lang, ts: Change error enum and message for wrong program ownership validation macro ([#1100](https://github.com/project-serum/anchor/pull/1154)).
```

Comment:
pr numbers dont match

Reply:
updated (first one was the issue number...)